app/pb_kits/playbook/pb_form/form_builder/date_picker_field.rb

```diff
@@ -8,6 +8,7 @@ def date_picker(name, props: {})
           prefix = @object_name
           html_attribute_name = "#{prefix}[#{name}]"
           html_id = "#{prefix}_#{name}"
+          props[:label] = @template.label(@object_name, name) if props[:label] && !props[:label].is_a?(String)
```

Comment:
We should treat true as the value that triggers this. That is how other inputs work, and a string value could be a user provided label, and this would take that away the ability to provide a prepared label value.
```suggestion
          props[:label] = @template.label(@object_name, name) if props[:label] == true
```

Reply:
Ah so the `==` operator type checks as well.  That's a much more elegant solution.  Didn't realize Ruby could do that.

My line 11 was checking if the value was 'truthy' and if it wasn't a String.  Which would preserve the 'ability to provide a prepared label value' and still allow a user to pass `true` for a Rails generated label.  But it doesn't account for all other variable types.  For example someone could pass an integer and it would just be registered as a `truthy` value and trigger the Rails generated label.  Thanks for catching this.